docs/reference/configuration/loadpoints.md

```diff
@@ -245,6 +245,9 @@ Definiert das Verhalten wann im PV Modus das Laden begonnen wird.
 #### `threshold`
 
 Definiert den Schwellenwert der Leistung am Netzanschlusspunkt in Watt (W).
+Wobei hierbei ein Wert bei "residualpower" in site berücksichtigt werden muss. 
```

Comment:
Ich würde die Hinweise in eine Info Box nach den "Möglichen Werten" stecken.

```
:::info
Ist für die evcc Site über den Parameter `residualPower` ein Versatz des Soll-Arbeitspunktes der Überschussregelung definiert, muss dieser Wert beim Setzen des `threshold` Wertes berücksichtigt werden.
Ist bspw. `residualpower` auf 200 gesetzt (die evcc Regelung setzt den Soll-Arbeitspunkt auf 200W Einspeisung), dann führt ein `enable` `threshold` Wert von 100 nicht dazu, dass ab 100W Netzbezug der Ladepunkt das Laden startet, sondern dass sich die verbleibende Einspeisung um 100W reduziert.
:::
```

Ich hab' versucht dein Beispiel etwas griffiger zu formulieren. Evtl. kannst du den 2ten Halbsatz, "sondern ..." noch ergänzen, ab wann bei der Einstellung dann konkret geladen wird.

Reply:
Die Idee mit der Info Box find ich gut. Was das weitere Bearbeiten angeht, bin ich mangels Wissen über Umgang mit Github, leicht überfordert.

Reply:
Einfach meinen Textvorschlag kopieren und an die richtige Stelle in deinem Fork kopieren, evtl. ergänzen und deinen git Fork nochmal synchronisieren, so dass deine Änderung hier im PR mit neuem Commit auftaucht ... :-)

Reply:
Hab zwar fast nix verstanden, aber scheint funktioniert zu haben. Ist ja ne Wissenschaft für sich ;-)  Nur das mit der Info Box krieg ich nicht hin.